### PR TITLE
[Doc] Remake Backup docs

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

File: docs/administration/Backup_and_restore.md [hunk 1]
```diff
@@ -2,37 +2,79 @@
...
-> - In each database, only one running BACKUP or RESTORE job is allowed each time.
+> - In each database, only one running BACKUP or RESTORE job is allowed each time. Otherwise, StarRocks returns an error.
 
 ## Back up data
 
-StarRocks supports full backup on granular level of database, table, or partition.
+StarRocks supports FULL backup on the granularity level of database, table, or partition.
 
-If you have stored a large quantity of data in a table, we recommend that you back up and restore data by partition to minimize the losses caused by possible job failures. If you need to back up data at a regular interval, you can strategize a partition plan (by a certain time interval, for example) for your table, and back up new partitions each time.
+If you have stored a large quantity of data in a table, we recommend that you back up and restore data by partition in case of possible job failures. If you need to back up data at a regular interval, you can strategize a [dynamic partitioning](../table_design/dynamic_partitioning.md) plan (by a certain time interval, for example) for your table, and only back up new partitions each time.
```
Comment (amber-create) [inline docs/administration/Backup_and_restore.md:16]: a large quantity of data：改为 -> a large amount of data

Comment (amber-create) [inline docs/administration/Backup_and_restore.md:16]: and only back up new partitions each time：这半句就是分区策略能够实现的目的，建议跟前半句融合；另外，only 修饰的主体不太对。建议改为 ->xxx, which enables/allows backing up only (the data from) new partitions at the specified interval.

File: docs/administration/Backup_and_restore.md [hunk 2]
```diff
@@ -58,8 +100,8 @@ You can restore the data snapshot backed up in the remote storage system to the
...
+> - Because data are backed up as snapshots, the data loaded after the snapshot is generated are not included in the snapshot. Therefore, if you load data into the old cluster after the snapshot is generated and before the RESTORE job is completed, you also need to load the data into the cluster that data is restored into. It is recommended that you load data into both clusters in parallel for a period of time after the data migration is complete, and then migrate your application to the new cluster after verifying the correctness of the data and services.
```
Comment (amber-create) [inline docs/administration/Backup_and_restore.md:103]: Because data are backed up as snapshots：主谓单复数不一致，data are -> data is

Comment (amber-create) [inline docs/administration/Backup_and_restore.md:103]: the data loaded after the snapshot is generated are not included in the snapshot：第一个 the snapshot 改为 -> a snapshot

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
 ## Description
 
-Creates a repository in a remote storage system that is used to store data snapshots for [Backup and restore data](../../../administration/Backup_and_restore.md).
+Creates a repository in a remote storage system that is used to store data snapshots for [backing up and restoring data](../../../administration/Backup_and_restore.md).
 
 > **CAUTION**
 >
-> Only root user and superuser can create a repository.
+> Only users with the ADMIN privilege can create a repository.
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:9]: the ADMIN privilege：admin 用户拥有的应该是很多（复数）privilege

File: docs/administration/Backup_and_restore.md (hunk 2 above)
Comment (EsoragotoSpirit) [reply docs/administration/Backup_and_restore.md:103]: snapshot is generated
data ... are not included

**Review by amber-create — DISMISSED** [review]

File: docs/administration/Backup_and_restore.md [hunk 4]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
 
 BACKUP is an asynchronous operation. You can check the status of a BACKUP job status using [SHOW BACKUP](../sql-reference/sql-statements/data-manipulation/SHOW%20BACKUP.md), or cancel a BACKUP job using [CANCEL BACKUP](../sql-reference/sql-statements/data-definition/CANCEL%20BACKUP.md).
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:116]: 基线英文里多了status,    You can check the status of a BACKUP job using

File: docs/administration/Backup_and_restore.md [hunk 5]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
-> - If the RESTORE job overwrites an existing database, table, or partition, the overwritten data cannot be restored after the job enters the COMMIT phase of the recovery job. If the RESTORE job fails or is canceled at this point, the data may be corrupted and inaccessible. In this case, you can only perform the RESTORE operation again and wait for the job to complete. Therefore, we recommend that you do not restore data by overwriting unless your are sure that the current data is no longer used. The overwrite operation will check metadata consistency between the snapshot and the existing database, table, or partition. If inconsistency is detected, the RESTORE operation cannot be performed.
+You can restore the data snapshot backed up in the remote storage system to the current or other StarRocks clusters to restore or migrate data.
 
 ### (Optional) Create the repository in the new cluster
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:122]: 冠词：Create a repository

File: docs/administration/Backup_and_restore.md [hunk 6]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
+| sr_member_backup | 2023-02-07-14-45-53-143 | OK     |
 +------------------+-------------------------+--------+
-1 row in set (0.01 sec)
+1 row in set (1.16 sec)
 ```
 
 ### Restore data via the snapshot
 
-Restore data snapshots in the remote storage system to current or other StarRocks clusters to restore or migrate data using [RESTORE](../sql-reference/sql-statements/data-definition/RESTORE.md).
+Restore data snapshots in the remote storage system to the current or other StarRocks clusters to restore or migrate data using [RESTORE](../sql-reference/sql-statements/data-definition/RESTORE.md).
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:144]: 优化建议：句子有点长，using放后面整个结构有点复杂，同时用祈使句可能不太好。You can use the [RESTORE](../sql-reference/sql-statements/data-definition/RESTORE.md) statement to restore data snapshots in the remote storage system to the current or other StarRocks clusters.

File: docs/administration/Backup_and_restore.md [hunk 7]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
 
 RESTORE is an asynchronous operation. You can check the status of a RESTORE job using [SHOW RESTORE](../sql-reference/sql-statements/data-manipulation/SHOW%20RESTORE.md), or cancel a RESTORE job using [CANCEL RESTORE](../sql-reference/sql-statements/data-definition/CANCEL%20RESTORE.md).
+
+## Configure BACKUP or RESTORE jobs
+
+You can optimize the performance of BACKUP or RESTORE jobs by modifying the following configuration items in the BE configuration file **be.conf**:
+
+| Configuration item      | Description                                                                             |
+| ----------------------- | -------------------------------------------------------------------------------- |
+| upload_worker_count     | The maximum number of threads for the upload tasks of BACKUP jobs on the BE node. Default: `1`. Increase the value of this configuration item to increase the concurrency of the upload task. |
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:166]: 冠词：on the BE node --- on a BE node，下面两处相同修改

File: docs/administration/Backup_and_restore.md [hunk 8]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
+
+## Usage notes
+
+- Only users with the ADMIN privilege can back up or restore data.
+- In each database, only one running BACKUP or RESTORE job is allowed each time. Otherwise, StarRocks returns an error.
+- Because BACKUP and RESTORE jobs occupy many resources of your StarRocks cluster, you can back up and restore your data while your StarRocks cluster is not heavily loaded.
+- StarRocks does not support specifying data compression algorithm for data backup.
+- Because data is backed up as snapshots, the data loaded upon snapshot generation is not included in the snapshot. Therefore, if you load data into the old cluster after the snapshot is generated and before the RESTORE job is completed, you also need to load the data into the cluster that data is restored into. It is recommended that you load data into both clusters in parallel for a period of time after the data migration is complete, and then migrate your application to the new cluster after verifying the correctness of the data and services.
+- Before the RESTORE job is completed, you cannot operate the table to be restored.
+- Primary Key table cannot be restored to a StarRocks cluster earlier than v2.5.
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:178]: tables 用复数，不然零冠词了

File: docs/administration/Backup_and_restore.md [hunk 9]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
+- You do not need to create the table to be restored in the new cluster before restoring it. The RESTORE job automatically creates it.
+- If there is an existing table that has a duplicated name with the table to be restored, StarRocks first checks whether or not the schema of the existing table matches that of the table to be restored. If the schemas match, StarRocks overwrites the existing table with the data in the snapshot. If the schema does match, the RESTORE job fails. You can either rename the table to be restored using the keyword `AS`, or delete the existing table before restoring data.
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:180]: If the schema does not match 少了 not

File: docs/administration/Backup_and_restore.md [hunk 10]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
+- If the RESTORE job overwrites an existing database, table, or partition, the overwritten data cannot be restored after the job enters the COMMIT phase of the RESTORE job. If the RESTORE job fails or is canceled at this point, the data may be corrupted and inaccessible. In this case, you can only perform the RESTORE operation again and wait for the job to complete. Therefore, we recommend that you do not restore data by overwriting unless you are sure that the current data is no longer used. The overwrite operation first checks metadata consistency between the snapshot and the existing database, table, or partition. If an inconsistency is detected, the RESTORE operation cannot be performed.
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:181]: the overwritten data cannot be restored after the job enters the COMMIT phase 删除后面的 of the RESTORE job. 有点重复

File: docs/administration/Backup_and_restore.md [hunk 11]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
+- During a BACKUP or a RESTORE job, StarRocks automatically backs up or restores the [Single-table Materialized view](../using_starrocks/Materialized_view-single_table.md) of the corresponding table. Currently, StarRocks does not support backing up the [Asynchronous Refresh Materialized view](../using_starrocks/Materialized_view.md). You can only back up the physical table of the materialized view, which cannot be used for query acceleration or query rewriting.
+- Currently, StarRocks does not support backing up the data of users, privileges and resource groups.
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:183]: backing up the data of users 可能会有误解，以为不能备份用户的数据，Currently, StarRocks does not support backing up the configuration data related to user accounts, privileges, and resource groups.

File: docs/administration/Backup_and_restore.md [hunk 12]
```diff
@@ -48,50 +115,69 @@ ON (sr_member);
...
+- If the RESTORE job overwrites an existing database, table, or partition, the overwritten data cannot be restored after the job enters the COMMIT phase of the RESTORE job. If the RESTORE job fails or is canceled at this point, the data may be corrupted and inaccessible. In this case, you can only perform the RESTORE operation again and wait for the job to complete. Therefore, we recommend that you do not restore data by overwriting unless you are sure that the current data is no longer used. The overwrite operation first checks metadata consistency between the snapshot and the existing database, table, or partition. If an inconsistency is detected, the RESTORE operation cannot be performed.
+- During a BACKUP or a RESTORE job, StarRocks automatically backs up or restores the [Single-table Materialized view](../using_starrocks/Materialized_view-single_table.md) of the corresponding table. Currently, StarRocks does not support backing up the [Asynchronous Refresh Materialized view](../using_starrocks/Materialized_view.md). You can only back up the physical table of the materialized view, which cannot be used for query acceleration or query rewriting.
```
Comment (evelynzhaojie) [inline docs/administration/Backup_and_restore.md:182]: 中文里面还有”暂不支持备份恢复视图“，这个信息要体现不？是不是单表物化视图还是能继续使用作为查询加速的？

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 13]
```diff
@@ -23,73 +23,81 @@ PROPERTIES ("key"="value", ...)
...
+| READ ONLY           | Create a read-only repository. Note that you can only restore data from a read-only repository. When creating the same repository for two clusters to migrate data, you can create a read-only repository for the new cluster and only grant it RESTORE permissions.|
 | repository_name     | Repository name.                                             |
 | repository_location | Location of the repository in the remote storage system.     |
-| PROPERTIES          | Username/password or access key/endpoint to the remote storage system. See [Examples](#examples) for more instructions. |
+| PROPERTIES          |The credential method for accessing the remote storage system. |
 
 **PROPERTIES**:
 
-StarRocks Supports creating repository in HDFS, S3, OSS, and COS.
+StarRocks Supports creating repository in HDFS, AWS S3, and Google GCS.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:33]: supports 大小写和 repository 单复数，StarRocks supports creating repositories

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 14]
```diff
@@ -23,73 +23,81 @@ PROPERTIES ("key"="value", ...)
...
 
-- For OSS：
-  - "fs.oss.accessKeyId"：Access Key ID used to log in OSS.
-  - "fs.oss.accessKeySecret"：Access Key Secret used to log in OSS.
-  - "fs.oss.endpoint"：Endpoint of the OSS storage.
+    - If you use IAM user-based credential (Access Key and Secret Key) to access AWS S3, you don't need to specify this parameter, and you need to specify "aws.s3.access_key", "aws.s3.secret_key", and "aws.s3.endpoint".
+    - If you use Instance Profile to access AWS S3, you need to set this parameter to `true` and specify "aws.s3.region".
+    - If you use Assumed Role to access AWS S3, you need to set this parameter to `true` and specify "aws.s3.iam_role_arn" and "aws.s3.region".
+  
+  - "fs.s3a.access.key": The Access Key ID that you can use to access the Amazon S3 bucket.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:46]: 这个可能忘记改了，aws.s3.access_key  下面的改为 aws.s3.secret_key

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 15]
```diff
@@ -23,73 +23,81 @@ PROPERTIES ("key"="value", ...)
...
+  - "fs.s3a.secret.key": The Secret Access Key that you can use to access the Amazon S3 bucket.
+  - "fs.s3a.endpoint": The endpoint that you can use to access the Amazon S3 bucket.
+  - "aws.s3.iam_role_arn": The ARN of the IAM role that has privileges on the AWS S3 bucket in which your data files are stored. If you want to use assumed role as the credential method for accessing AWS S3, you must specify this parameter. Then, CelerData will assume this role when it analyzes your Hive data by using a Hive catalog.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:49]: CelerData 改为 StarRocks

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by EsoragotoSpirit** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**